Scripts/linux/dtllabservice.py

```diff
@@ -332,55 +332,128 @@ def getVirtualMachine(self, subscriptionId, vmId=None, name=None):
         Returns:
             The virtual machine that matches the provided filter, or None.
 
-        Example retun data:
+        Example return data:
```

Comment:
Is there any way to get the service to give you the expected output so we don't have to update this every time they change?